### PR TITLE
Don't use strlen() when we just need to check if string is empty or not.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Mailer/Message.php [hunk 1]
```diff
@@ -1610,7 +1610,7 @@ protected function encodeString(string $text, string $charset): string
      */
     protected function wrap(?string $message = null, int $wrapLength = self::LINE_LENGTH_MUST): array
     {
-        if ($message === null || strlen($message) === 0) {
+        if ($message === null || $message === '') {
```
Comment (othercorey) [inline src/Mailer/Message.php:1613]:
```suggestion
        if (empty($message)) {
```

Comment (markstory) [reply src/Mailer/Message.php:1613]: I would be careful with empty() as it also eats '0'.

Comment (othercorey) [reply src/Mailer/Message.php:1613]: True, but $message is `string|null`. But was just a suggestion.

Comment (ADmad) [reply src/Mailer/Message.php:1613]: So it can be string `0`, empty() gobble that up.

Comment (othercorey) [reply src/Mailer/Message.php:1613]: Sorry I missed it was a string lol.

**Pull request merged**